### PR TITLE
Jason new

### PR DESCRIPTION
**Pull request by jhdaguitera**
New commit. Added indicators with SMA algorithm

### CODE REVIEW

File: lean.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
+<<<<<<< HEAD
```
Comment (scheffeltravis) [inline lean.json:2]: Looks like you have some conflicts here.

File: sma/main.py [hunk 2]
```diff
@@ -6,12 +6,30 @@
...
 class SMAAlgorithm(QCAlgorithm):
-    def __init__(self):
+   # def __init__(self):
+    def Initialize(self):
         self.moving_averages_close = []
         self.moving_averages_high = []
         self.moving_averages_low = []
         self.ma_lengths = [5, 10, 20, 62]
         self.trade_status = len(self.ma_lengths) * [0]
+<<<<<<< HEAD
```
Comment (scheffeltravis) [inline sma/main.py:16]: Same thing here.

File: sma/example.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
+using QuantConnect.Data;
```
Comment (scheffeltravis) [inline sma/example.cs:1]: This file was actually deleted/renamed to `reference.cs`.

**Review by scheffeltravis — COMMENTED** [review]
I think you may need to merge the changes currently in `main` with your local branch, as it looks like there are some issues around old commits missing and conflicts.

**Comment by jhdaguitera** [comment]
hey im going to close this PR and resubmit

**Pull request closed without merging**